core/unit_test/TestTeamVector.hpp

```diff
@@ -864,6 +864,88 @@ class TestTripleNestedReduce {
 
 #endif
 
+template <typename ExecutionSpace, bool bInclusive, class Reducer>
+void checkScan() {
+  static constexpr int n = 1000000;
+  using size_type        = typename TEST_EXECSPACE::size_type;
+  using value_type       = typename Reducer::value_type;
+
+  Kokkos::View<value_type[n], TEST_EXECSPACE> inputs("inputs");
+  Kokkos::parallel_for(
+      Kokkos::RangePolicy<TEST_EXECSPACE>(0, n),
+      KOKKOS_LAMBDA(size_type i) { inputs(i) = i * 1. / n; });
+
+  static constexpr int nTeamThreadRange   = 1e3;
+  static constexpr int nThreadVectorRange = 1e2;
+  static constexpr int nPerTeam = nTeamThreadRange * nThreadVectorRange;
+  static constexpr int nTeams   = n / nPerTeam;
+
+  value_type result;
+  Reducer reducer(result);
+
+  Kokkos::View<typename Reducer::value_type[n], ExecutionSpace> outputs(
+      "outputs");
+
+  // run ThreadVectorRange parallel_scan
+  Kokkos::TeamPolicy<ExecutionSpace> policy(nTeams, Kokkos::AUTO, Kokkos::AUTO);
+  const std::string label =
+      (bInclusive ? std::string("inclusive") : std::string("exclusive")) +
+      std::string("Scan") + std::string(typeid(Reducer).name());
```

Comment:
You don't absolutely have to cast `typeid(Reducer).name()` when you concatenate.